tutorials/rag/evaluate_rag_cicd.py

```diff
@@ -0,0 +1,159 @@
+""" Evaluate RAG pipeline `ask_question` using Humanloop, with Prompt stored in code."""
+
+import argparse
+import os
+from dotenv import load_dotenv
+import inspect
+import uuid
+import pandas as pd
+from chromadb import chromadb
+from openai import OpenAI
+from humanloop import Humanloop
+from cicd.cicd import check_evaluation_results, create_new_evaluation, run_evaluation
+from cicd.utils import populate_template
+
+
+# Init clients and setup Vector db
+load_dotenv()
+chroma = chromadb.Client()
+openai = OpenAI(api_key=os.getenv("OPENAI_KEY"))
+humanloop = Humanloop(
+    api_key=os.getenv("HUMANLOOP_KEY"), base_url="https://neostaging.humanloop.ml/v5"
```

Comment:
base url.

also this client instantiation is repeated above. probably not good for it to be repeated?